src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -243,6 +292,33 @@ def _agentpool_format_group(item):
     ])
 
 
+def _connected_registry_format_group(item):
+    parent_id = _get_value(item, 'parent', 'id')
+    parent_name = '' if parent_id.isspace() else parent_id.split('/connectedRegistries/')[1]
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('MODE', _get_value(item, 'mode')),
+        ('CONNECTION STATE', _get_value(item, 'connectionState')),
+        ('PARENT', parent_name),
+        ('LOGIN SERVER', _get_value(item, 'loginServer', 'host')),
+        ('LAST ACTIVITY (UTC)', _get_value(item, 'lastActivityTime')),
+        ('LAST SYNC (UTC)', _get_value(item, 'parent', 'syncProperties', 'lastSyncTime')),
+        ('SYNC SCHEDULE', _get_value(item, 'parent', 'syncProperties', 'schedule')),
+        ('SYNC WINDOW', _get_value(item, 'parent', 'syncProperties', 'syncWindow'))
+    ])
+
+
+def _connected_registry_list_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('MODE', _get_value(item, 'mode')),
+        ('CONNECTION STATE', _get_value(item, 'connectionState')),
+        ('PARENT', _get_value(item, 'parent_name')),
+        ('LOGIN SERVER', _get_value(item, 'loginServer_host')),
+        ('LAST SYNC (UTC)', _get_value(item, 'parent_syncProperties_lastSyncTime'))
```

Comment:
@rosanch Why do we not also include `LAST ACTIVITY (UTC)` here as well?

Reply:
We had a meeting to discuss what should be displayed. But I'll bring this up  in next meeting.